src/components/header/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./style.css";
+
+export const Header: React.FC = () => {
```

Comment:
talvez um pouco mais específico aqui?
```suggestion
export const PageHeader: React.FC = () => {
```
ou
```suggestion
export const SectionHeader: React.FC = () => {
```
o que acha?

Reply:
Bem melhor haha

Fiquei com o `SectionHeader` 👍🏻